bib/cmd/bootc-image-builder/main.go

```diff
@@ -604,11 +597,11 @@ func buildCobraCmdline() (*cobra.Command, error) {
 	rootCmd.AddCommand(versionCmd)
 
 	rootCmd.AddCommand(manifestCmd)
-	manifestCmd.Flags().Bool("tls-verify", true, "require HTTPS and verify certificates when contacting registries")
+	manifestCmd.Flags().Bool("tls-verify", false, "require HTTPS and verify certificates when contacting registries")
```

Comment:
Should we also put a big "DEPRECATED" here like we did for "local" ?

Reply:
Ah flip dropped the commit and re-added and forgot to add this back in. Thanks!